boards/snps/nsim/arc_v/doc/index.rst

```diff
@@ -0,0 +1,201 @@
+  usage here: :ref:`here <toolchain_designware_arc_mwdt>`.
+
+.. note::
+   Note that even if both GNU and MWDT toolchain support is declared for the target some tests or
+   samples can be only built with either GNU or MWDT toolchain due to some features limited to a
+   particular toolchain.
+
+Use this configuration to run basic Zephyr applications and kernel tests in
+nSIM, for example, with the :zephyr:code-sample:`synchronization` sample:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/synchronization
+   :host-os: unix
+   :board: nsim_arc_v/rmx100
+   :goals: flash
+
+This will build an image with the synchronization sample app, boot it using
+nSIM, and display the following console output:
+
+.. code-block:: console
+
+      *** Booting Zephyr OS build zephyr-v3.2.0-3948-gd351a024dc87 ***
+      thread_a: Hello World from cpu 0 on nsim_arc_v!
+      thread_b: Hello World from cpu 0 on nsim_arc_v!
+      thread_a: Hello World from cpu 0 on nsim_arc_v!
+      thread_b: Hello World from cpu 0 on nsim_arc_v!
+      thread_a: Hello World from cpu 0 on nsim_arc_v!
+
+
+.. note::
+   To exit the simulator, use :kbd:`Ctrl+]`, then :kbd:`Ctrl+c`
+
+.. _board_nsim_arc_v_verbose_build:
+
+.. tip::
+   You can get more details about the building process by running build in verbose mode. It can be
+   done by passing ``-v`` flag to the west: ``west -v build -b nsim_hs samples/synchronization``
+
+Debugging
+=========
+
+.. _board_nsim_arc_v_debugging_gdb:
+
+Debugging with GDB
+------------------
+
+.. note::
+   Debugging on nSIM via GDB is only supported on single-core targets (which use standalone
+   nSIM).
+
+.. note::
+   The normal ``west debug`` command won't work for debugging applications using nsim boards
+   because both the nSIM simulator and the debugger use the same console for
+   input / output.
+   In case of GDB debugger it's possible to use a separate terminal windows for GDB and nSIM to
+   avoid intermixing their output.
+
+After building your application, open two terminal windows. In terminal one, use nSIM to start a GDB
+server and wait for a remote connection with following command:
+
+.. code-block:: console
+
+   west debugserver --runner arc-nsim
+
+In terminal two, connect to the GDB server using RISC-V GDB. You can find it in Zephyr SDK:
+
+* you should use :file:`riscv64-zephyr-elf-gdb`
+
+This command loads the symbol table from the elf binary file, for example the
+:file:`build/zephyr/zephyr.elf` file:
+
+.. code-block:: console
+
+   riscv64-zephyr-elf-gdb  -ex 'target remote localhost:3333' -ex load build/zephyr/zephyr.elf
+
+Now the debug environment has been set up, and it's possible to debug the application with gdb
+commands.
+
+Modifying the configuration
+***************************
+
+If modification of existing nsim configuration is required or even there's a need in creation of a
+new one it's required to maintain alignment between
+
+* Zephyr OS configuration
+* nSIM & MDB configuration
```

Comment:
MDB? Shouldn't this be lldbac or so?

Reply:
I'm not pretty sure about lldbac, so I just removed MDB reference here